wiki/fr/fr-Central-Servers.md

```diff
@@ -0,0 +1,24 @@
+---
+layout: wiki
+title: "Serveurs centraux"
+lang: "fr"
+permalink: "/wiki/Central-Servers"
+---
+
+# Enregistrement du serveur public
+
+En mai 2020, afin de minimiser le problème de la saturation du serveur central, nous avons créé plusieurs serveurs centraux basés sur le genre musical (chacun pouvant lister 150 serveurs au maximum). 
```

Comment:
"_À partir de mai 2020_", rather than "_En mai 2020_".

Good use of "saturation".

There is no use of "_serveur central_" in the English wording, hence it's not useful to add that in the French translation.

**Summary**: À partir de mai 2020, afin de minimiser le problème de saturation du serveur, nous avons créé des serveurs centraux basés sur le genre (chacun avec une limite actuelle de 150 serveurs).

Reply:
Sounds goods for me.